### PR TITLE
DCM-45: Support Weekly Thursday period type in Automation

### PR DESCRIPTION
**Pull request by Piumal1999**
## The issue I worked on
See [https://issues.openmrs.org/browse/DCM-45](https://issues.openmrs.org/browse/DCM-45) 

## Description of what I changed:
Added Weekly Thursday period support to automation

### CODE REVIEW

File: api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java [hunk 1]
```diff
@@ -51,6 +51,13 @@ public void transformToDHISPeriod() {
 		Assert.assertEquals("20171108", sdf.format(startDate.getTime()));
 		Assert.assertEquals("20171114", sdf.format(endDate.getTime()));
 		Assert.assertEquals(period, "2017WedW46");
+
+		//test weekly thursday, date now at 2017/nov/14
+		startDate = (Calendar) endDate.clone();
+		period = Context.getService(DHISConnectorService.class).transformToDHISPeriod(startDate, endDate, "WeeklyThursday", null);
```
Comment (HerbertYiga) [inline api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java:57]: thanks @Piumal1999  have you also tested with a running instance of openmrs

Comment (Piumal1999) [reply api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java:57]: Yeah @HerbertYiga

**Comment by akshika47** [comment]
@jayasanka-sack would you be able to take look at this PR?

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Piumal1999 !

**Pull request merged**